### ISSUE TITLE
Loss Nan

### ISSUE THREAD

**Issue by yk-pku** [Limingxing00/RDE-VOS-CVPR2022#2]
Hi, I load the pretrain s0 model of 'model_s0_final.pth', and train the stage 3 without using BL30K, the loss is Nan, How can I fix this ?

**Comment by Limingxing00** [comment]
Hi, sorry to reply you so late.

I'm already working and rarely look at the previous repo.

If you use mixed precision operations, you may encounter very large loss scales during training. But don't worry, it usually takes a few iterations to get back to normal.

**Comment by yk-pku** [comment]
Thanks for the response, I find it does not affect the training actually. Please feel free to close this issue~
